Source/Core/VideoCommon/AVIDump.cpp

```diff
@@ -136,7 +136,7 @@ bool AVIDump::CreateVideoFile()
     s_codec_context->codec_tag =
         MKTAG('X', 'V', 'I', 'D');  // Force XVID FourCC for better compatibility
   s_codec_context->codec_type = AVMEDIA_TYPE_VIDEO;
-  s_codec_context->bit_rate = 400000;
+  s_codec_context->bit_rate = SConfig::GetInstance().m_DumpFramesBitrate * 1000;
```

Comment:
Can't really test that on Windows until #4957 is merged, since it fixes the bug with bit-rate settings not having any effect.